### DISCUSSION TITLE
Please Help with 404 Issue on Vercel Deployment - SolidStart App

### DISCUSSION THREAD

**Discussion by Kingrashy12** [solidjs/solid-start#1137] [Q&A] [answered]
I have successfully deployed my SolidStart app to Vercel, but when I access the deployed URL, I'm encountering a 404 error. The deployment process seemed smooth, but I'm having trouble figuring out why the app is not loading as expected.

**Comment by wakaztahir** [comment]
You should give here, deployment URL, information about how you deployed your solid start app (ssg / ssr / adapter) & information about 404 (try opening network tab in chrome, see if all the resources are not loading)

**Reply by Kingrashy12** [reply]
https://pingspace.vercel.app/

**Reply by wakaztahir** [reply]
This didn't help, Provide Vite config, Also provide commands you use to build & deploy...

**Reply by Kingrashy12** [reply]
import solid from "solid-start/vite";
import { defineConfig } from "vite";

export default defineConfig({
  plugins: [
    solid({
      ssr: true,
      solid: {
        generate: "ssr",
      },
    }),
  ],
});

**Reply by wakaztahir** [reply]
This doesn't contain any deploy info...

**Reply by Kingrashy12** [reply]
what are the deployment info

**Reply by Kingrashy12** [reply]
This my first time using solidstart

**Reply by wakaztahir** [reply, marked as answer]
You should install this package, Set it up in vite config like this `plugins: [solid({ adapter: vercel() })]`

https://www.npmjs.com/package/solid-start-vercel

Also read its package details

**Reply by Kingrashy12** [reply]
Ok

**Reply by Kingrashy12** [reply]
Thanks alot it's fixed
